sky/tools/gn

```diff
@@ -48,7 +48,13 @@ def to_gn_args(args):
 
     gn_args['is_debug'] = args.debug
     gn_args['is_clang'] = args.clang and args.target_os not in ['android', 'fnl']
-    gn_args['flutter_develop_mode'] = args.develop
+
+    if args.develop:
+        gn_args['dart_runtime_mode'] = 'develop'
+    elif args.debug:
+        gn_args['dart_runtime_mode'] = 'profile'
```

Comment:
I see, this is for /gn.  Yeah, this seems fine.  We could also just have the flag be --profile. :)  But this is fine too.

Happy to help wire up the bots to build this.